GitUI/UserControls/RevisionGrid/Columns/AvatarColumnProvider.cs

```diff
@@ -50,7 +54,38 @@ public override void OnCellPainting(DataGridViewCellPaintingEventArgs e, GitRevi
 
             int imageSize = e.CellBounds.Height - _padding - _padding;
 
-            Task<Image?> imageTask = _avatarProvider.GetAvatarAsync(revision.AuthorEmail, revision.Author, imageSize);
+            Task<Image?> imageTask;
+
+            void GetAvatar() =>
+                _lastAvatar = imageTask = _avatarProvider.GetAvatarAsync(revision.AuthorEmail, revision.Author, imageSize);
+
+            if (_email == revision.AuthorEmail && _author == revision.Author)
+            {
+                imageTask = _lastAvatar;
+                if (imageTask.Status == TaskStatus.RanToCompletion)
+                {
+                    // Manage exceptional case where cached image have been cleaned by user
+                    try
+                    {
+#pragma warning disable VSTHRD002 // Avoid problematic synchronous waits
+                        if (imageTask.Result.PixelFormat == System.Drawing.Imaging.PixelFormat.DontCare)
+#pragma warning restore VSTHRD002 // Avoid problematic synchronous waits
```

Comment:
❔
```suggestion
                        Image? image = await imageTask;
                        if (image?.PixelFormat is System.Drawing.Imaging.PixelFormat.DontCare)
```

Reply:
I don't think it's possible as we can't make that method async (as it is an overrided method).
And I don't think complexify the code to handle it another way doesn't worth it...